products/linuxmint.md

```diff
@@ -4,6 +4,9 @@ category: os
 tags: linux-distribution
 iconSlug: linuxmint
 permalink: /linuxmint
+alternate_urls:
+-   /linux-mint
+-   /mint
```

Comment:
```suggestion
```

Mint is also the name of another software, see https://duckduckgo.com/?t=ftsa&q=mint+software&atb=v396-1&ia=web..